WordPress/Classes/ViewRelated/Blog/My Site/NoSitesViewModel.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+struct NoSitesViewModel {
```

Comment:
We could add tests for this VM quite quickly I think.

Reply:
Done! [b2e5be3](https://github.com/wordpress-mobile/WordPress-iOS/pull/21456/commits/b2e5be320b69b45518734dc33dc9d89e460baf8a)